src/runway.rs

```diff
@@ -715,10 +847,12 @@ pub(crate) async fn run<H, D, MK, DP, SH>(
         unit_messages_for_network: runway_io.unit_messages_for_network,
         ordered_batch_rx,
         resolved_requests: runway_io.resolved_requests,
+        starting_round_sender,
         node_ix: config.node_ix,
         session_id: config.session_id,
         n_members: config.n_members,
         max_round: config.max_round,
+        salt: 0,
```

Comment:
Shouldn't this be... random or something?

Reply:
Fixed